server/models/Event.js

```diff
@@ -9,30 +9,6 @@ const Event = sequelize.define('event', {
   },
   name: DataTypes.STRING,
   data: DataTypes.JSONB,
-
```

Comment:
?

Reply:
You don't need to define these, the `belongsTo` call takes care of it